package.json

```diff
@@ -11,11 +11,12 @@
   "license": "MIT",
   "repository": "koajs/trie-router",
   "dependencies": {
-    "methods": "1",
+    "errors": "^0.3.0",
```

Comment:
@jonathanong sorry, tad bit confused. "same error" as in what's defined in this PR (using `errors` module) or an http error like 404 / what was already happening (see #19)?